api/services/protocols/local.js

```diff
@@ -25,7 +25,7 @@ const crypto    = require('crypto');
  */
 exports.register = async function (req, res, next) {
   const email    = req.param('email'),
-    name = req.param('name'),
+    name = req.param('username'),
```

Comment:
Yeah, I forgot to change the tests. I meant to figure out when this was originally changed because the front end used to work. I'll figure that out later.